dataSeedingScript.js

```diff
@@ -0,0 +1,63 @@
+const mongoose = require('mongoose');
```

Comment:
Wish I knew more about mongo to be able to give you feedback here. From what little I know, this looks great!